### PR TITLE
add null check for values when getting invalid tag names

### PR DESCRIPTION
**Pull request by warren-ovo**

### CODE REVIEW

**Review by prashantsinghovo — APPROVED** [review]

File: packages/datadog-metrics-tracker/src/base.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export abstract class MetricsTracker {
   private getInvalidTagNames(tags: { [name: string]: string }, measurementName: string) {
     try {
       return Object.entries(tags)
-        .filter(([_, value]) => value.length === 0)
+        .filter(([_, value]) => value?.length === 0)
```
Comment (zoe-j-m) [inline packages/datadog-metrics-tracker/src/base.ts:66]: Should we have a test to cover this?

**Review by zoe-j-m — COMMENTED** [review]
OK, with a comment - happy for it to be addressed in a future PR

**Review by zoe-j-m — APPROVED** [review]

**Review by manikant16 — COMMENTED** [review]
Please upgrade pkg version as well otherwise it won't reflect the changes

**Review by manikant16 — APPROVED** [review]
Please upgrade pkg version as well otherwise it won't reflect the changes

File: packages/datadog-metrics-tracker/src/base.ts (hunk 1 above)
Comment (warren-ovo) [reply packages/datadog-metrics-tracker/src/base.ts:66]: yup and I'm working on it now, just getting help from Mani in setting up the environment for testing. I was thinking that it's critical and small enough to push the bugfix first and then create the new tests to cover it. What do you think?

**Pull request merged**